syntax = "proto3";

package test.app;
option go_package = "test.app";

import "google/protobuf/wrappers.proto";
import "google/protobuf/any.proto";

service TestService {
    rpc ProcessTick(stream OrderResponse) returns(stream OrderRequest);
}

message OrderRequest {
    int64 order_id = 1;
    int32 client_id = 2;
}

message OrderResponse {
    int64 order_id = 1;
    int32 client_id = 2;
}

